eve/workers/pod-linter/Dockerfile

```diff
@@ -20,8 +21,28 @@ RUN dnf install -y git \
   ShellCheck \
   && dnf clean all \
   && adduser -u 1042 --home /home/eve eve \
-  && chown eve:eve /home/eve/workspace \
+  && chown -R eve:eve /home/eve \
   && pip install tox \
   && pip install buildbot-worker==${BUILDBOT_VERSION}
 
+# Install Go tooling to lint the Kubernetes Operators.
```

Comment:
Given we seem to be creating (almost) the same container image twice, does it really make sense to make a distinction between a `linter` and `builder` container?

Reply:
I think the reason behind the split is that we want to use `centos:7` as base image (and that's what we do for the pod-builder).

Now, the issue with pod-linter is that we need a not too old `shellcheck`, which is not available on CentOS 7 so Jordi switched the base image to `fedora:29` for this pod (IIRC it was too cumbersome to get a recent `shellcheck` on CentOS)

That's the reason behind the split I think (keep using CentOS 7 everywhere, except for linting).